internal/flink/command_artifact_create.go

```diff
@@ -132,10 +132,15 @@ func (c *command) createArtifact(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	if err := utils.UploadFile(resp.GetUploadUrl(), artifactFile, resp.GetUploadFormData()); err != nil {
-		return err
+	if strings.ToLower(cloud) == "azure" {
```

Comment:
what happens if you provide the cloud as aws, and resp.GetUploadUrl() is a azure link?

Reply:
Then it wont work. So the link contains information like cloud provider, which bucket name you are uploading to, and expiration time. If any of this is invalid, the upload will fail